README.md

```diff
@@ -1,6 +1,6 @@
 # napari-brainglobe-io
 
-[![License](https://img.shields.io/pypi/l/brainglobe-napari-io.svg?color=green)](https://github.com/napari/brainglobe-napari-io/raw/master/LICENSE)
+[![License](https://img.shields.io/pypi/l/brainglobe-napari-io.svg?color=green)](https://github.com/brainglobe/brainglobe-napari-io/raw/master/LICENSE)
```

Comment:
```suggestion
[![License](https://img.shields.io/pypi/l/brainglobe-napari-io.svg?color=green)](https://github.com/brainglobe/brainglobe-napari-io/blob/main/LICENSE)
```

This link will point to the rendered version of the LICENSE file currently at master, rather than the raw version.

The previous link was broken since GitHub changed how to link to raw files, but if we want to link to the raw file instead, the link is below
https://raw.githubusercontent.com/brainglobe/brainglobe-napari-io/main/LICENSE